internal/vpn/server.go

```diff
@@ -29,15 +30,25 @@ type Server struct {
 
 // NewServer creates VPN server instance.
 func NewServer(cfg ServerConfig, l logrus.FieldLogger) (*Server, error) {
+	var defaultNetworkIfc string
 	s := &Server{
 		cfg:   cfg,
 		log:   l,
 		ipGen: NewIPGenerator(),
 	}
 
-	defaultNetworkIfc, err := netutil.DefaultNetworkInterface()
-	if err != nil {
-		return nil, fmt.Errorf("error getting default network interface: %w", err)
+	if cfg.NetworkInteface == "" {
+		defaultNetworkIfcs, err := netutil.DefaultNetworkInterface()
+		if err != nil {
+			return nil, fmt.Errorf("error getting default network interface: %w", err)
+		}
+		netIfcs, isMultiple := s.checkingNetworkInterface(defaultNetworkIfcs)
+		if isMultiple {
+			return nil, fmt.Errorf("multiple default network interface detected, please set once in setting or be single: %v", netIfcs)
```

Comment:
"multiple default network interfaces detected...set a default one for VPN server or remove one: %v"

Reply:
Maybe 

`ifcs, hasMultipe := s.hasMutipleInterfaces(...)`

Reply:
Done.